src/pages/cart/main/Card.jsx

```diff
@@ -0,0 +1,61 @@
+import { useCart } from "contexts/CartContext";
+import { useWishlist } from "contexts/WishlistContext";
+
+export const Card = (cart) => {
+  const { dispatch } = useWishlist();
```

Comment:
you can name this dispatch something like wishlistDispatch, so it will avoid the confusion when you will be having multiple dispatches